python/taichi/lang/kernel.py

```diff
@@ -415,7 +416,9 @@ def call_back():
 
                             return call_back
 
-                        assert has_torch and isinstance(v, torch.Tensor)
+                        assert has_torch
+                        import torch
```

Comment:
We have to `import torch` within the function because it might not be available. `torch` is conditionally imported in https://github.com/taichi-dev/taichi/blob/c93a62fa78f2763e0a8b7104bd6b1dd7741d6c8c/python/taichi/lang/util.py#L11-L14, but previously it got included in this module due to `from .util import *`.